templates/config.jinja

```diff
@@ -2,6 +2,12 @@ temporalGrpcAddress: {{ TEMPORAL_ADDRESS | default("temporal-k8s:7233") }}
 port: {{ TEMPORAL_UI_PORT | default("8080") }}
 enableUi: {{ TEMPORAL_UI_ENABLED | default("true") }}
 defaultNamespace: {{ TEMPORAL_DEFAULT_NAMESPACE | default("default") }}
+workflowTerminateDisabled: {{ TEMPORAL_WORKFLOW_TERMINATE_DISABLED | default("false") }}
+workflowCancelDisabled: {{ TEMPORAL_WORKFLOW_CANCEL_DISABLED | default("false") }}
+workflowSignalDisabled: {{ TEMPORAL_WORKFLOW_SIGNAL_DISABLED | default("false") }}
+workflowResetDisabled: {{ TEMPORAL_WORKFLOW_RESET_DISABLED | default("false") }}
+batchActionsDisabled: {{ TEMPORAL_BATCH_ACTIONS_DISABLED | default("false") }}
+hideWorkflowQueryErrors: {{ TEMPORAL_HIDE_WORKFLOW_QUERY_ERRORS | default("false") }}
```

Comment:
General question: What is the use case for having the default value also in the Jinja file as well as the `config.yaml` ? It seems as though this will require updating defaults in 2 places instead of 1.

Reply:
I was wondering this too, but I figured the Jinja default would just catch cases where no value was passed in for whatever reason so the Juju default should always be passed in and therefore always take precedence. But @kelkawi-a can elaborate.

Reply:
Yup you're right, I've removed the unnecessary defaults from the Jinja file and kept the config defaults instead. Thanks!